TFlippy_TerritoryControl_Core_Dev/Entities/Equipment/Equipment.as

```diff
@@ -469,16 +272,16 @@ void onDie(CBlob@ this)
 		}
 		if (this.get_string("equipment_torso") != "")
 		{
-			if(this.get_string("equipment_torso") == "bulletproofvest")
+			if(this.get_string("equipment_torso") == "bulletproofvest" || this.get_string("equipment_torso") == "keg")
 			{ }
-			else if(this.get_string("equipment_torso") == "suicidevest" && !this.exists("vest_explode"))
-				server_CreateBlob(this.get_string("equipment_torso"), this.getTeamNum(), this.getPosition());
-			else if(this.get_string("equipment_torso") != "suicidevest")
+			else if(!this.exists("vest_explode"))
 				server_CreateBlob(this.get_string("equipment_torso"), this.getTeamNum(), this.getPosition());
 		}
 		if (this.get_string("equipment_boots") != "")
 		{
-			server_CreateBlob(this.get_string("equipment_boots"), this.getTeamNum(), this.getPosition());
+			if(this.get_string("equipment_boots") == "combatboots")
+			{ }
```

Comment:
is this suppose to be empty?

Reply:
I'd assume goldenguy wanted to do
if (this.get_string("equipment_boots") != "combatboots")